ecommerce/README.md

```diff
@@ -1,23 +1,9 @@
-### Core
-
-It contains functions, classes and constants to be used as a basis in the core.
-
-### Customers
-
-It contains customer model and address model for customers.
-
-### Products
-
-It contains product model, category model, stock model and price model for products.
-
-### Baskets
-
-It contains basket model and basket item in basket model.
-
-### Orders
-
-It contains billing address model, shipping address model, order bank account model, order model and order item model.
-
-### Payments
-
-It contains bank model and bank account model.
\ No newline at end of file
+## All models' serializers, filters, viewsets was created, these are;
```

Comment:
- I am a bit confused here, because these kind of information should'nt be added on README.md file, because you can alrady mention this infos in your commit and when you create a PR you can leave these information as a comment